### PR TITLE
Cortex XDR Device Control Violations

### PR DESCRIPTION
**Pull request by EliorKedar**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31327
## Description
Cortex XDR device control violations - added as part of a new API endpoint.
 - playbook for fetching DC violations.
 - Response playbook  DC violations.
 - XDR Incident handling v3 - DC violation enrichment.
 
## Screenshots
Cortex_XDR_device_control_violations
JOB_-_Cortex_XDR_query_endpoint_device_control_violations

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_README.md [hunk 1]
```diff
@@ -2,21 +2,22 @@ This playbook is triggered by fetching a Palo Alto Networks Cortex XDR incident.
...
+After the remediation, if there are no new alerts, the playbook stops the alert sync and closes the XDR incident and investigation. For performing the bidirectional sync, the playbook uses the incoming and outgoing mirroring feature added in XSOAR version 6.0.0. After the `Calculate Severity - Generic v2` sub-playbook’s run, Cortex XSOAR will be treated as the single source of truth for the severity field, and it will sync only from Cortex XSOAR to XDR, so manual changes for the severity field in XDR will not update in the XSOAR incident.
 
 ## Dependencies
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 * Entity Enrichment - Generic v3
-* Palo Alto Networks - Hunting And Threat Detection
+* PANW - Hunting and threat detection by indicator type V2
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_README.md:12]: Looks like the Readme still has the old name

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations.yml [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+      script: '|||ad-get-user'
+      type: regular
+      version: -1
+    taskid: e384e86d-82bc-45ea-8706-24bb237d8deb
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 447.5,
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations.yml:323]: Line between task 5 and 6 dont look good

**Review by David-BMS — CHANGES_REQUESTED** [review]
@EliorKedar Update the playbook images they still have typo's

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations_README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| MessageSubject | The subject of the message for communication with the involved users. | Device control violation | Optional |
+| MessageBody | The body of the message for communication with the involved users. | Hello,<br/>Your user was involved with a device control violation. Please open the following link to fill in the needed information to understand the incident further. | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Cortex XDR device control violations](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations_README.md:39]: Update link to image

File: Packs/CortexXDR/Playbooks/playbook-JOB_-_Cortex_XDR_query_endpoint_device_control_violations_README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+| Severity | The severity of the created incident when the device control events were found.<br/>Valid values are; <br/>   0 - Unknown<br/>   0.5 - Informational<br/>   1 - Low<br/>   2 - Medium<br/>   3 - High<br/>   4 - Critical | 1 | Optional |
+| IncidentType | The desired incident type for the created incident when the device control violations were found. | Cortex XDR Device Control Violations | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![JOB - Cortex XDR query endpoint device control violations](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-JOB_-_Cortex_XDR_query_endpoint_device_control_violations_README.md:47]: Update link to image

File: Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_README.md [hunk 5]
```diff
@@ -34,20 +35,21 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
...
+| TimeStamp | Timestamp in relative date format for query device control events from Cortex XDR | 10 days | Optional |
 
 ## Playbook Outputs
 ---
 There are no outputs for this playbook.
 
 ## Playbook Image
 ---
-![Cortex XDR incident handling v3](https://raw.githubusercontent.com/demisto/content/d0bde5eac9154adc18d49814a73a6e44a69313b2/Packs/CortexXDR/doc_files/Cortex_XDR_incident_handling_v3.png)
+![Cortex XDR incident handling v3](Insert the link to your image here)
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_README.md:55]: Update link to new image

**Comment by ShirleyDenkberg** [comment]
@David-BMS @michalgold Doc review completed

**Review by David-BMS — APPROVED** [review]
@EliorKedar Please update the readme images for all 3 playbooks and add the link and description to configure the job. Besides that approved.

File: Packs/CortexXDR/Layouts/layoutscontainer-Cotrex_XDR_Device_Control_Violations.json [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+        "tabs": [
+            {
+                "id": "summary",
+                "name": "Legacy Summary",
+                "type": "summary"
+            },
+            {
+                "hidden": false,
+                "id": "zmjvpqxkcs",
+                "name": "XDR Device Control Violations",
```
Comment (David-BMS) [inline Packs/CortexXDR/Layouts/layoutscontainer-Cotrex_XDR_Device_Control_Violations.json:12]: @EliorKedar Also please move the layout here
image

**Comment by ShirleyDenkberg** [comment]
@David-BMS @michalgold Added the Active Directory Query v2 integration to the list of integrations needed in the playbook-Cortex_XDR_device_control_violations_README.  
Doc review completed.

**Pull request merged**